File: MGCDDemo/MGCDDemo/ViewController.m
//
//  ViewController.m
//  MGCDDemo
//
//  Created by mal on 2019/9/25.
//  Copyright © 2019 mal. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
{
    dispatch_semaphore_t _sem;
}

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    _sem = dispatch_semaphore_create(1);
}


@end

File: AIInPutView/AIInPutView/ViewController.swift
//
//  ViewController.swift
//  AIInPutView
//
//  Created by mal on 2020/7/6.
//  Copyright © 2020 mal. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    lazy var inputTextView: AIInputTextView = {
        let view = AIInputTextView.viewWith()
        view.maxLines = 5
        view.placeHolder = "我是占位的文本"
        return view
    }()
    
    override func viewDidLoad() {
        super.viewDidLoad()
    }
    
    @IBAction func onShowInputViewBtnClick(_ sender: UIButton) {
        inputTextView.show(inView: self.view)
    }
}

